nuxeo-coldstorage/src/main/resources/OSGI-INF/coldstorage-security.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.coldstorage.security">
+    <require>org.nuxeo.ecm.core.security.defaultPermissions</require>
+    <extension target="org.nuxeo.ecm.core.security.SecurityService"
+               point="permissions">
+        <permission name="WriteColdStorage">
+            <include>ReadWrite</include>
```

Comment:
I think we should ask @jaubenque if we can state that `WriteColdStorage` means `ReadWrite` too

Reply:
I see the "WriteColdStorage" as an atomic permission, so that customers can decide in which group of permissions to add it.
Can you confirm it's the case ?
And can you explain why do you include the ReadWrite into it ?
Thanks

Reply:
> I see the "WriteColdStorage" as an atomic permission, so that customers can decide in which group of permissions to add it. Can you confirm it's the case ?

We see it as permission. Like Read or Write. Here we weren't able to provide this permission on UI part. I mean how customer will provide it into users.  

> And can you explain why do you include the ReadWrite into it ?

It's here for questioning :). As `WriteColdStorage`  will be enough to make the move, as you cannot read or write. If we need to take it separately we have to provide the three ones.

Reply:
@RSalem07
`WriteColdStorage` will be enough to move the document into cold storage.
I did some tests this morning

Reply:
> @RSalem07 and @jaubenque
> `WriteColdStorage` will be enough to move the document into cold storage.
> I did some tests this morning

Was the user a non-administrator? When I opened [this ticket](https://jira.nuxeo.com/browse/NXP-30293) it was because when I set the user permission to `WriteColdStorage`, he couldn't move the document into cold storage.

Reply:
In [NXP-28600](https://jira.nuxeo.com/browse/NXP-28600) it is mentioned that it should be added to the `Write` role though, and not `ReadWrite`.

Reply:
I have applied read permission also.
And not only WriteColdStorage

Reply:
hello,

following my discussion with Julien, we will add the `ReadWrite` by default when we add the `WriteColdStorage` @doulba is the initial thing that you did :)